usaspending_api/common/helpers/generic_helper.py

```diff
@@ -113,12 +108,27 @@ def convert_matview_to_view(matview_sql):
     return sql
 
 
-def generate_matviews(materialized_views_as_traditional_views=False):
+def get_temp_matview_sql_files_dict(matview_dir: Path):
+    temp_sql_files = [
+        {"name": key, "sql_file": matview_dir / val["sql_filename"]} for key, val in MATERIALIZED_VIEWS.items()
+    ]
+    temp_sql_files += [
+        {"name": key, "sql_file": matview_dir / val["sql_filename"]} for key, val in CHUNKED_MATERIALIZED_VIEWS.items()
+    ]
+    return temp_sql_files
+
+
+def generate_matviews(materialized_views_as_traditional_views=False, parallel_worker_id: str = None):
     with connection.cursor() as cursor:
         cursor.execute(CREATE_READONLY_SQL)
         cursor.execute(DEPENDENCY_FILEPATH.read_text())
-        subprocess.call(f"python3 {MATVIEW_GENERATOR_FILE} --dest {DEFAULT_MATIVEW_DIR} --quiet", shell=True)
-        for matview_sql_lookup in TEMP_SQL_FILES:
+
+        matview_dir = DEFAULT_MATIVEW_DIR
+        if parallel_worker_id:
```

Comment:
Can we make this if statement less implicit? Change it to `if parallel_worker_id is not None`

Reply:
As it stands its more implicit than it needs to be.

Reply:
It covers two scenarios, if the param is `None` or if the param comes in as empty string. So while slightly implicit it is comprehensive and less verbose. It's a common idiom in python for checking a string (or e.g. list) has value to it, so I think worth leaving as is.